src/main/resources/templates/emails/student.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <title th:remove="all">Template for HTML email with inline image</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  </head>
+  <body>
+    <p>welcome</p>
+   <a th:href="@{http://localhost:8080/api/update-student(studentId=${student.studentId})}">click here to confirm</a>
```

Comment:
Never hard code URL in a template, they should come from properties file

Reply:
I fixed